mobilizon_reshare/storage/db.py

```diff
@@ -17,7 +19,18 @@ def __init__(self, path: Path):
         if not self.is_init:
             self.path.parent.mkdir(parents=True, exist_ok=True)
 
+    async def __implement_db_changes__(self):
+        # return
+        print('implementing db changes')
```

Comment:
rimuovi e usa il logger a livello INFO, magari con un messaggio un po' più chiaro tipo "Updating database to latest migration

Reply:
fatto